tsfresh/examples/har_dataset.py

```diff
@@ -47,7 +47,7 @@ def download_har_dataset():
     
     zipurl = 'https://github.com/MaxBenChrist/human-activity-dataset/blob/master/UCI%20HAR%20Dataset.zip?raw=true'
 
-    if os.path.exists(data_file_name):
+    if os.path.exists(data_file_name_dataset):
```

Comment:
can you change this to if os.path.exists(data_file_name_dataset) and if os.path.exists(data_file_name_classes) to make sure that both datasets and classes files are downloaded?